src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/probation/User.kt

```diff
@@ -1,11 +1,13 @@
 package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation
 
 data class User(
-  val id: Long? = null,
-  val code: String? = null,
-  val name: Name? = null,
-  val teams: List<Detail>? = null,
-  val username: String?,
+  val id: Long,
+  val code: String,
+  val name: Name,
+  val teams: List<TeamDetail> = listOf(),
+  val provider: Detail,
+  val username: String? = null,
   val email: String? = null,
-  val unallocated: Boolean? = null,
+  val telephoneNumber: String? = null,
+  val unallocated: Boolean,
```

Comment:
This had to be change to resemble what it actually was 
```
data class Staff(
    val id: Long,
    val code: String,
    val name: Name,
    val teams: List<Team>,
    val provider: Provider,
    val username: String?,
    val email: String?,
    val telephoneNumber: String?,
    val unallocated: Boolean,
)
```
In CVL for some reason the object became multiple forms forms unrelated structures. Hence all the ?